Part2-API-Schemas/openapi.yaml

```diff
@@ -206,6 +203,53 @@ components:
         - properties: 
             derivedFrom: 
               $ref: "https://api.swaggerhub.com/domains/Plattform_i40/Part1-MetaModel-Schemas/V3.0.1#/components/schemas/Reference"
+    AssetLink:
+      properties:
```

Comment:
add description, e.g.

`description: A list of AssetLinkds. An AssetLink may either represent a SpecificAssetIds or the globalAssetId of the asset represented by the Asset Administration Shell. The usage of the attributes externalSubjectId and semanticId in the context of querying asset IDs is deprecated, i.e. it is not recommended to use them for querying.`

Reply:
I have added it to the V3.1.0 file.

Reply:
which PR?

Reply:
Will create the PR, seems I only have it locally right now. My bad.

Reply:
Here it is: #293